### ISSUE TITLE
[kokoro] Only run snapshot tests on Xcode 9.2+

### ISSUE THREAD

**Issue by romoore** [material-components/material-components-ios#6327]
iOS 11.2 isn't supported on Xcodes below 9.2.  We shouldn't execute snapshot tests there.

```
Executing tests from //components/TextFields:snapshot_tests
-----------------------------------------------------------------------------
2019-01-14 20:23:57,142 Will consider the test as test type xctest to run.
2019-01-14 20:23:57,263 Generating xctestrun file.
2019-01-14 20:23:57,698 Failed to import biplist module. Will use tool /usr/libexec/PlistBuddy to handle the binary format plist.
Traceback (most recent call last):
  File "/usr/local/Cellar/python/2.7.14/Frameworks/Python.framework/Versions/2.7/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/local/Cellar/python/2.7.14/Frameworks/Python.framework/Versions/2.7/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "../xctestrunner/file/downloaded/__main__.py", line 265, in <module>
  File "../xctestrunner/file/downloaded/__main__.py", line 259, in main
  File "../xctestrunner/file/downloaded/__main__.py", line 200, in _SimulatorTest
  File "../xctestrunner/file/downloaded/__main__.py", line 159, in _RunSimulatorTest
  File "../xctestrunner/file/downloaded/__main__/xctestrunner/simulator_control/simulator_util.py", line 311, in CreateNewSimulator
xctestrunner.shared.ios_errors.IllegalArgumentError: The simulator os version 11.2 is not supported. Supported simulator os versions are [u'8.1', u'8.2', u'8.3', u'8.4', u'9.0', u'9.1', u'9.2', u'9.3', u'10.0', u'10.1', u'10.2', u'10.3', u'11.0', u'11.1', u'11.1'].
```

**Comment by romoore** [comment]
Will take a look at this tonight.

**Comment by romoore** [comment]
Looks like the simplest way forward is to switch to iOS 11.1. It's available in kokoro's Xcode 9.0 environments.
